jobs_detector/main.py

```diff
@@ -20,14 +31,133 @@ def jobs_detector():
 @click.option('-c', '--combinations', type=str,
               callback=lambda _, x: x.split(',') if x else x)
 def hacker_news(post_id, keywords, combinations):
-    """
-    This subcommand aims to get jobs statistics by parsing "who is hiring?"
-    HackerNews posts based on given set of keywords.
-    """
-    # HINT: You will probably want to use the `BeautifulSoup` tool to
-    # parse the HTML content of the website
-    pass
+    
+    data_to_parse = request_jobs_page(post_id)
+    job_posts = retrieve_job_postings(data_to_parse) # get all top comments
+    
+
+    # renaming this from response, I found it confusing because we already use response as a variable in the get URL and data request.
+    job_stats = []
+    
+    job_stats.append(get_total_jobs(job_posts))
```

Comment:
`get_total_jobs` returns a `string` data type object and we want `job_stats` to be a list.